ProcessLib/HeatTransport/CreateHeatTransportProcess.cpp

```diff
@@ -0,0 +1,96 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "CreateHeatTransportProcess.h"
+
+#include "HeatTransportProcess.h"
+#include "HeatTransportProcessData.h"
+
+namespace ProcessLib
+{
+namespace HeatTransport
+{
+std::unique_ptr<Process> createHeatTransportProcess(
+    MeshLib::Mesh& mesh,
+    Process::NonlinearSolver& nonlinear_solver,
+    std::unique_ptr<Process::TimeDiscretization>&& time_discretization,
+    std::vector<ProcessVariable> const& variables,
+    std::vector<std::unique_ptr<ParameterBase>> const& parameters,
+    BaseLib::ConfigTree const& config)
+{
+    //! \ogs_file_param{process__type}
+    config.checkConfigParameter("type", "HEAT_TRANSPORT");
+
+    DBUG("Create HeatTransportProcess.");
+
+    // Process variable.
+    auto process_variables = findProcessVariables(
+        variables, config,
+        {//!
+         //\ogs_file_param_special{process__HEAT_TRANSPORT__process_variables__process_variable}
```

Comment:
These comments must not be broken up into two lines. Otherwise Doxygen won't recognize them later on.